packages/pf3-component-mapper/package.json

```diff
@@ -48,8 +48,8 @@
     "node-sass": "^4.12.0",
     "patternfly-react": "^2.28.2",
     "prop-types": "^15.6.2",
-    "react": "^16.6.3",
-    "react-dom": "^16.6.3",
+    "react": "^16.13.0",
```

Comment:
Oh, we probably need to keep this low to achieve compatibility.

Reply:
Eslint `5.x` --> `6.x` is okey?